Makefile

```
@@ -9,49 +9,54 @@ comma := ,
 BUILDDIR = _build
 COMPILER = xelatex
 BIBCOMPILER = biber
-PROJECT = cv
 BIBLIOGRAPHY = bibliography
 
-pdf: headon clean
+cv-pdf coverletter-pdf: headon clean
```

```suggestion
pdf: headon clean
```

This should stay pdf, as the two names do not actually change the execution